crates/forge/tests/cli/main.rs

```diff
@@ -18,6 +18,7 @@ mod doc;
 mod eip712;
 mod geiger;
 mod inline_config;
+mod inspect;
```

Comment:
hmmm I think it would be best to not add this module here, but we don't have a `cli::zk` path either, so probably in `zk_cmd`, or `zk_inspect` to do like `zk_build`

but yeah point is, `inspect` here would look like an "upstream" test and not ours